core/trino-main/src/test/resources/hydra/ssl.cnf

```diff
@@ -0,0 +1,19 @@
+[ req ]
+default_bits       = 4096
+distinguished_name = req_distinguished_name
```

Comment:
Why are these different than in `core/trino-main/src/test/resources/cert/localhost.conf `